### ISSUE TITLE
Add repo labels

### ISSUE THREAD

**Issue by BNAndras** [exercism/pyret#99]
@ErikSchierboom, I'd like to accept community contributions for this track. Can you add the 'community-contributions-accepted' label? There's no pause contributions GitHub action to disable here. I'd also like to add the `pyret` label while you're in there. Thanks in advance.

**Comment by ErikSchierboom** [comment]
@BNAndras If you add them here: https://github.com/exercism/pyret/blob/main/.appends/.github/labels.yml, once the PR is merged you should get another PR to update https://github.com/exercism/pyret/blob/main/.github/labels.yml, and when that is merged you'll have the labels :)

**Comment by BNAndras** [comment]
Sorry, I didn't mean the labels for issues. I wanted those labels added to the About corner where the exercism-track one is.

![image](https://github.com/exercism/pyret/assets/20251272/7fa7b990-fbb5-48f3-98a8-ac72c99d1f49)

**Comment by ErikSchierboom** [comment]
Right. Done.
